scripts/createdb.sh

```diff
@@ -41,7 +41,6 @@ elif
     echo -e "\n\n\033[32m$HELP_STRING\033[0m\n"
 else
     USERNAME_DBNAME=$1
-    dropdb $1
-    createuser -P -s -e $USERNAME_DBNAME
-    createdb --username=$USERNAME_DBNAME --owner=$USERNAME_DBNAME -W $USERNAME_DBNAME
+    PGPASSWORD=postgres createuser -U postgres -w -P -s -e $USERNAME_DBNAME
+    PGPASSWORD=$USERNAME_DBNAME createdb --username=$USERNAME_DBNAME --owner=$USERNAME_DBNAME -w $USERNAME_DBNAME
 fi
```

Comment:
What does this do again? Drop DB before creating if it exists? Please confirm the best practice.

Reply:
So, I deleted the `createdb.sh` script because I discovered that the `setup.sh` scripts creates the database and installs all required packages. If this script creates the database, I see no need for the `createdb.sh` script. Am I right?